src/deploy/auto_common/Scp.py

```diff
@@ -28,21 +28,21 @@ def get_file(adress, user, passw, localfile, remotefile):
         client = paramiko.SSHClient()
         client.load_system_host_keys()
         client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        trans = paramiko.Transport((adress, 22))
-        trans.connect(username=user, password=passw)
-        sftp = paramiko.SFTPClient.from_transport(trans)
+        client.connect(hostname=adress, username=str(user), password=str(passw))
+        sftp=client.open_sftp()
         sftp.get(remotefile, localfile)
         sftp.close()
-        trans.close()
+        client.close()
 
     @staticmethod
     def put_file(adress, user, passw, localfile, remotefile):
+        #sftp.put(localfile, remotefile)
```

Comment:
This line is a  commented dupe of line #45.  Is it needed or can it be removed?

Reply:
removed